### PR TITLE
Fix #475 — Error when trying to create an empty directory.

### LINKED ISSUE #475: Error while trying to generate a configuration file without subfolder with CLI.
```
$ kinto --ini kinto.ini

Traceback (most recent call last):
  File "~/.virtualenvs/kinto/bin/kinto", line 9, in <module>
    load_entry_point('kinto', 'console_scripts', 'kinto')()
  File "~/mozilla/kinto/kinto/__main__.py", line 72, in main
    init(config_file, backend)
  File "~/mozilla/kinto/kinto/config/__init__.py", line 50, in init
    render_template("kinto.tpl", config_file, **values)
  File "~/mozilla/kinto/kinto/config/__init__.py", line 14, in render_template
    os.makedirs(folder)
  File "~/.virtualenvs/kinto/lib/python2.7/os.py", line 157, in makedirs
    mkdir(name, mode)
OSError: [Errno 2] No such file or directory: ''
```

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

**Comment by leplatrem** [comment]
no test but r+ :)

**Comment by Natim** [comment]
Let me write one :+1:

File: kinto/tests/test_config.py [hunk 1]
```diff
@@ -97,3 +97,33 @@ def test_init_memory_values(self, mocked_render_template):
...
+        config.render_template('kinto.tpl', destination, **{
+            'secret': "abcd-ceci-est-un-secret",
+            'storage_backend': 'cliquet.storage.memory',
+            'cache_backend': 'cliquet.cache.memory',
+            'permission_backend': 'cliquet.permission.memory',
+            'storage_url': '',
+            'cache_url':  '',
+            'permission_url': ''
+        })
+        self.assertTrue(os.path.exists('/tmp/config/kinto.ini'))
```
Comment (leplatrem) [inline kinto/tests/test_config.py]: `destination` no?

Comment (Natim) [inline kinto/tests/test_config.py]: Yes sorry.

**Pull request merged**